vnames.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Add a comment about the purpose of this file at the top.

Reply:
Can't add comments to json file but the README has a section that illustrates what this file is used for.

Reply:
ok, good enough